src/util/permission/hasPermissionLevelRole.ts

```diff
@@ -0,0 +1,22 @@
+import { GuildMember } from "discord.js";
+import { guildConfigs } from "../../guild/config/guildConfigs";
+import getRoleTypeById from "./getRoleTypeById";
+import { PermissionLevel } from "./permissionLevel";
+import resolvePermissionLevelFromRoleType from "./resolvePermissionLevelFromRoleType";
+
+export function hasPermissionLevelRole(permissionLevel: PermissionLevel, member: GuildMember) {
+    let allRoles: string[] = [];
```

Comment:
Forgot to include this comment from the copied file
`    // We want to get a list of every significant role id from every guild`
above L8

No reason to remove it